examples/screenshot.c

```diff
@@ -213,7 +201,9 @@ static void write_image(const char *filename, int width, int height) {
 			exit(EXIT_FAILURE);
 		}
 		close(fd[0]);
-		execlp("convert", "convert", "-depth", "8", "-size", size, "rgba:-",
+		// We requested WL_SHM_FORMAT_XRGB8888 in little endian, so that's BGRA
+		// in big endian.
+		execlp("convert", "convert", "-depth", "8", "-size", size, "bgra:-",
```

Comment:
Doesn't this come from ImageMagick? Does that need to be added to the dependencies for the library?

Reply:
No, it is a runtime dependency. It's also a dependency specific to this example.